modules/@apostrophecms/ui/ui/apos/components/AposCheckbox.vue

```diff
@@ -30,13 +30,21 @@
         :size="10"
       />
     </span>
-    <span
+    <div
       v-if="choice.label"
       :class="{'apos-sr-only': field.hideLabel }"
       class="apos-choice-label-text"
     >
-      {{ $t(choice.label) }}
-    </span>
+      <span>
+        {{ $t(choice.label) }}
+      </span>
+      <!-- eslint-disable vue/no-v-html -->
+      <p
+        v-if="(choice.help || choice.htmlHelp)"
+        class="apos-choice-label-text--help apos-field__help"
+        v-html="$t(choice.help || choice.htmlHelp)"
```

Comment:
Choice `help` should not be rendered as HTML unlike `htmlHelp`. I think you need two separate entries and checks for this render.

Reply:
This code was carried over from [AposInputWrapper.vue#105](https://github.com/apostrophecms/apostrophe/blob/1b84327ab0d4952ea800321da0076da2b045e946/modules/%40apostrophecms/schema/ui/apos/components/AposInputWrapper.vue#L105).